src/vellum/workflows/tests/test_dataset_row.py

```diff
@@ -126,3 +126,92 @@ def test_dataset_row_with_empty_inputs():
 
     assert serialized_dict["label"] == "test_label"
     assert serialized_dict["inputs"] == {}
+
+
+def test_dataset_row_with_dict_inputs():
```

Comment:
keep this test, and add one wherever we are testing sandbox.py validating that we can run with a `inputs_dict` like we are below